partcad/src/partcad/part_factory.py

```
@@ -33,18 +33,20 @@ def __init__(
 
     def _create_part(self, config: object) -> Part:
-        part = Part(config)
-        part.project_name = self.target_project.name  # TODO(clairbee): pass it via the constructor
-        # TODO(clairbee): Make the next line work for part_factory_file only
+        part = Part(self.target_project.name, config)
         part.instantiate = lambda part_self: self.instantiate(part_self)
         part.info = lambda: self.info(part)
         part.with_ports = self.with_ports
         return part
 
     def _create(self, config: object):
```

Ruff (0.8.2)

Missing return type annotation for private function `_create`

Add return type annotation: `None`

(ANN202)